include/NovelRT.Interop/NrtErrorHandling.h

```diff
@@ -27,6 +31,9 @@ extern "C"
     void Nrt_setErrMsgIsInvalidOperationInternal();
     void Nrt_setErrMsgIsCharacterNotFoundInternal();
     void Nrt_setErrMsgIsOutOfMemoryInternal();
+    void Nrt_setErrMsgKeyNotFound();
+    void Nrt_setErrMsgRuntimeNotFound();
+    void Nrt_setErrMsgErrorUnknown();
```

Comment:
Same here for the `Internal` suffix:
```suggestion
    void Nrt_setErrMsgKeyNotFoundInternal();
    void Nrt_setErrMsgRuntimeNotFoundInternal();
    void Nrt_setErrMsgErrorUnknownInternal();
```